source/index.html.slim

```diff
@@ -38,25 +38,26 @@ title: Home
           p
             ' It's being held on Thursday 2 November (workshops) and Friday 3 November (conference) at Te Papa on the Wellington waterfront.
         .body-inner-wrapper__section
+          h2 🎤 I want to speak
+          p Our CFP for speakers is now open!
```

Comment:
Even though this information is also at Papercall I think it's worth putting the date the CFP closes here - it's also an indication to people who might not want to submit a proposal about when other stuff is likely to happen